### DISCUSSION TITLE
WD smartscreen block GHelpe.exe - nothing happens.

### DISCUSSION THREAD

**Discussion by Kikimahe** [seerge/g-helper#1445] [General]
Hi, 

I don't know if I'm alone, but recantly for me GHelper.exe is totally blocked by Windows Defender "Smartscreen".

When Ghelper is closed, if I double click on Ghelper.exe, nothing happens.
There is no popup alert window to warn me that the software is not signed (to run it anyway).

To run Ghelper.exe I must either :
- Disable Smartscreen in Windows 11
- Or restart windows
- Or go to the task scheduler and execute the Ghelper task.

I don't know what's going on.

**Comment by seerge** [comment]
Hello, I can only quote FAQ here

> [Windows Defender marks app as malware / virus](https://github.com/seerge/g-helper#windows-defender-marks-app-as-malware--virus)
False positives from Windows Defender (or any other similar system that uses machine learning for detection) is possible as the application is not digitally signed with a certificate. You can always download a version below or compile the app by yourself. All application sources are open and can be monitored from A to Z :)

**Reply by Kikimahe** [reply]
Yes @Seerge, I read it correctly and I understand all of this. :-)

But my problem is different :
1) If I quit Ghelper and run Ghelper.exe, nothing happens, ghelper does not launch.
2) This popup has disappeared, this one allows me to “Run anyway”
![renditionDownload](https://github.com/seerge/g-helper/assets/142881210/9e0c8593-23ab-4f9f-b3ff-e0000c73e0e3)


I can run Gelper.exe only by Task Scheduler or by turning off Smartscreen. Then Ghelper.exe launches.

It's strange.

**Reply by seerge** [reply]
Right click on exe file, select Properties -> click Unblock there

**Reply by Kikimahe** [reply]
Nothing to unblock in properties. ;'( 

But i fix it with reinstalling windows 11.  ;-)
